mscore/network/loginmanager.cpp

```diff
@@ -373,7 +373,7 @@ static void clearHttpCacheOnRenderFinish(QWebEngineView* webView)
       QWebEnginePage* page = webView->page();
       QWebEngineProfile* profile = page->profile();
 
-      // workaround for the crashes sometimes happend in Chromium on macOS with Qt 5.12
+      // workaround for the crashes sometimes happenning in Chromium on macOS with Qt 5.12
```

Comment:
one n too much

Reply:
yikes, thanks. I'll update to codespell dictionary to catch this. e2ca31e